src/python-common/ceph/deployment_utils/drive_group.py

```diff
@@ -0,0 +1,140 @@
+import fnmatch
```

Comment:
I think that `deployment_utils` is a new directory, but it is tricky to verify with the Github UI. If that is the case, I would suggest not calling it 'deployment_utils' because then you are forced to add non-util things like exceptions to it *or* you would need to add another module if you have lots of exceptions like deployment_exceptions. By naming the module 'deployment' then you can have`deployment/utils.py` and `deployment/exceptions.py`, which will make it easier in the future to expand and organize without being tied to '_utils.py'

Reply:
`src/python-common` itself is a new addition. There is the opportunity to rename or restructure everything right now. The idea was to not add this to `src/pybind`, as it should be something new. Something wich can absorb old oddities, like `ceph_argparse.py` or `ceph_daemon.py` into a proper python package.